script/ProcessQueueExitsandRewards.s.sol

```diff
@@ -0,0 +1,97 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.19;
+
+import "forge-std/Script.sol";
+import { ITinyMevEth } from "src/interfaces/ITinyMevEth.sol";
+import { BatchScript } from "forge-safe/BatchScript.sol";
+
+interface IMevEthQueue is ITinyMevEth {
+    function processWithdrawalQueue(uint256 newRequestsFinalisedUntil) external;
+    function requestsFinalisedUntil() external returns (uint256);
+    function queueLength() external returns (uint256);
+    function withdrawalQueue(uint256 ticketNumber) external returns (bool claimed, address receiver, uint128 amount, uint128 accumulatedAmount);
+    function claim(uint256 ticketNumber) external;
+    function addOperator(address) external;
+    function operators(address) external returns (bool);
+}
+
+/// @notice script to process withdraw queue
+contract ProcessQueueExitsandRewardsScript is BatchScript {
+    error NothingToProcess();
+
+    function run(uint256 numExits) public {
+        address safe = 0x617c8dE5BdE54ffbb8d92716CC947858cA38f582;
+        IMevEthQueue mevEth = IMevEthQueue(0x24Ae2dA0f361AA4BE46b48EB19C91e02c5e4f27E);
+
+        uint256 exitsBalance = numExits * 32 ether;
+        // assuming rewards balance = remaing multisig balance - 10%
+        uint256 rewardsBalance = (safe.balance - exitsBalance) * 90 / 100;
```

Comment:
protocol treasury safe: `0xe664B134d96fdB0bf7951E0c0557B87Bac5e5277`
should transfer fee in this script too

Reply:
I was thinking the same thing. Only way to do it consistently is to transfer 10% of `rewardsToProcess`. Then it can't double spend on leftover rewards not used in the queue processing.

Reply:
Updated:
https://github.com/manifoldfinance/mevETH2/pull/273/commits/e821ea6c909f6474a3fa9076bba90900a6f2e5ce